.github/workflows/build.yml

```diff
@@ -70,16 +67,6 @@ jobs:
             ${{ runner.os }}-build-${{ env.cache-name }}-
             ${{ runner.os }}-build-
             ${{ runner.os }}-
-      - name: Setup TurboRepo
-        # Conditionally setup turborepo
-        # In the past, turborepo would silently ignore empty environment variables
-        # This is no longer the case, so we need to check if the secret is set
-        # You cannot use `if: ${{ secrets.TURBO_TOKEN != '' }}` because secrets are not available in the `if` condition
-        if: ${{ env.SECRETS_TURBO_TOKEN != '' }}
-        run: |
-          echo "TURBO_TOKEN=${{ secrets.TURBO_TOKEN }}" >> $GITHUB_ENV
-          echo "TURBO_TEAM=${{ secrets.TURBO_TEAM }}" >> $GITHUB_ENV
-          echo "TURBO_REMOTE_ONLY=${{ secrets.TURBO_REMOTE_ONLY }}" >> $GITHUB_ENV
```

Comment:
For now I don't see us making many changes to this, so decided to remove this so the action does not error.

Longterm, I do think using the Turbo repo remote cache will speed up our builds. This requires us to have a Vercel account + a paid plan if we exceed their free tier.

Looks like the free tier is 10 GB of artifact downloads per month. For the next level it's $10 per user per month + we get 10 GB + $0.50 per GB after that.

We may also be able to use the Github aciton cache action as well + possibly get a better deal...need to research more if we're interested.